data/test/vtgate/dml_cases.txt

```diff
@@ -1193,6 +1193,117 @@
   }
 }
 
+# update with no where clause
+"update user_extra set val = 1"
+{
+  "Original": "update user_extra set val = 1",
+  "Instructions": {
+    "Opcode": "UpdateScatter",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "update user_extra set val = 1",
+    "Table": "user_extra"
+  }
+}
+
+
+# update with non-comparison expr
+"update user_extra set val = 1 where id between 1 and 2"
+{
+  "Original": "update user_extra set val = 1 where id between 1 and 2",
+  "Instructions": {
+    "Opcode": "UpdateScatter",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "update user_extra set val = 1 where id between 1 and 2",
+    "Table": "user_extra"
+  }
+}
+
+# update with primary id through IN clause
+"update user_extra set val = 1 where user_id in (1, 2)"
+{
+  "Original": "update user_extra set val = 1 where user_id in (1, 2)",
```

Comment:
This should ideally be its own opcode, like `UpdateIN`, but it can be done as an improvement later.